### DISCUSSION TITLE
Quarkus Newsletter #28 - January

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#32415] [Quarkus Blog/Website]
# Quarkus Newsletter #28 - January

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-newsletter-28/

**Comment by sergeyk777** [comment]
Alpaca stock photos can add a unique touch to various projects. One great resource I'd recommend is Unsplash. They have a diverse collection of high-resolution <a href="https://depositphotos.com/vectors/alpaca.html">alpaca</a> images contributed by talented photographers from around the world. It's an excellent option if you're looking for free images and want to avoid licensing issues.

Additionally, if you're willing to invest a bit, Shutterstock and Adobe Stock are fantastic paid platforms with extensive libraries. They offer a wide variety of alpaca photos, often with different styles and themes to choose from. Depending on your project's budget, these options can provide the professional quality you need.
